back/app/services/projects_finder_service.rb

```diff
@@ -0,0 +1,95 @@
+class ProjectsFinderService
+  def initialize(projects, current_user = nil, params = {})
+    @projects = projects
+    @current_user = current_user
+    @params = params
```

Comment:
I would be specific about the params passed in ie `page_size` & `page_number`, makes it clearer what's needed

Reply:
Good idea! Thanks.